pkg/catalogv2/system/system.go

```diff
@@ -284,6 +286,17 @@ func (m *Manager) isInstalled(namespace, name, version string, values map[string
 			return false, err
 		}
 
+		if minVersion != "" {
+			min, err := semver.NewVersion(minVersion)
+			if err != nil {
+				return false, err
+			}
+			if min.LessThan(ver) || min.Equal(ver) {
+				logrus.Infof("Skipping installing/upgrading desired version %v for release %v, current version %v is greater or equal to minimal required version %v", desired.String(), name, ver.String(), minVersion)
+				return true, nil
+			}
+		}
+
 		if (desired.LessThan(ver) || desired.Equal(ver)) && equality.Semantic.DeepEqual(values, release.Config) {
```

Comment:
The new logic is not considering the situation where we have changed the values file.  If the values has changed, but the version is sufficient, we should do an upgrade to the exact same version installed but with new values.